src/ol_infrastructure/lib/consul.py

```diff
@@ -1,12 +1,17 @@
 import base64
 import json
-import textwrap
+
 import yaml
 
 
 def build_consul_userdata(env_name):
```

Comment:
This is going to need to be updated to accept a list of environment names for the WAN peers, and the autojoin_wan file should only be written out if it is a non-`None` value.